pkg/database/config.go

```diff
@@ -96,6 +96,50 @@ func (c *Config) ConnectionString() string {
 	return strings.Join(p, " ")
 }
 
+// clone creates a deep copy of the configuration.
```

Comment:
I worry about forgetting to update this if we ever add a field to `Config`. Thoughts about using a deepcopy library? Maybe github.com/jinzhu/copier